syntax = "proto3";

package pb;

service Microservice {
    rpc GetMark(Marks.GetRequest) returns (Marks.GetResponse) {}
}

message Marks {
    message GetRequest {
        float areasAmountPer100k = 1;
        float sportsAmountPer100k = 2;
        float areasSquarePer100k = 3;
        float subwayDistance = 4;
        float pollutionPercentage = 5;
    }

    message GetResponse {
        float mark = 1;
    }
}
